ext/storage_sources/s3_store/s3_log_system.cpp

```diff
@@ -0,0 +1,67 @@
+#include <aws/core/Aws.h>
+#include "s3_log_system.h"
+#include <cstdarg>
+
+S3LogSystem::S3LogSystem(WT_EXTENSION_API *wtApi, uint32_t wtVerbosityLevel)
+{
+    // If the verbosity level is out of range it will default to AWS SDK Error level.
```

Comment:
np: Should comments be formatted like this instead:
```/* If the verbosity level is out of range it will default to AWS SDK Error level. */```

Reply:
Please see Tammy's comment above. 

> Please see the wiki for [General Guidelines for Commenting Code](https://github.com/mongodb/mongo/wiki/Style-Guidelines), and the [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html#Comments). The // style is generally preferred for C++ code, but the important thing is to be consistent.

I think we should be using C++ commenting style for code in our C++ files (that we have decided to orient with C++ styling over C) so I would like to maintain these comments in this way. We can maintain consistency by reformatting our past and future comments this way too.